test/test.js

```diff
@@ -186,15 +187,15 @@ function checkForGene(test, frequency, input) {
 function warningsCountError(input, frequency) {
   var sheet = xlsx.parse(input),
       network = spreadsheetController.parseSheet(sheet),
-      warningsCountErrorArray = network.errors.filter(function(x){return x.errorCode == "WARNINGS_OVERLOAD";});
+      warningsCountErrorArray = network.errors.filter(function(x){return x.errorCode === "WARNINGS_OVERLOAD";});
```

Comment:
One best practice in collaborative coding: when the group has adopted a particular style (and it has), as you evolve the existing code and see opportunities to improve style adherence, go do it. This is one such opportunity: we have decided upon individualized variable declarations, so this block of code should become:

    var sheet = xlsx.parse(input);
    var network = spreadsheetController.parseSheet(sheet);
    var warningsCountErrorArray = network.errors.filter(function (x) { return x.errorCode === "WARNINGS_OVERLOAD"; });